tests/foreman/api/test_ansible.py

```diff
@@ -59,3 +63,65 @@ def test_fetch_and_sync_ansible_playbooks(target_sat):
     )
     assert task_details[0].result == 'success'
     assert len(task_details[0].output['result']['created']) == playbooks_count
+
+
+@pytest.mark.e2e
+@pytest.mark.no_containers
+@pytest.mark.rhel_ver_list([7, 8, 9])
```

Comment:
can we include fips enabled hosts as well here?

Reply:
Addressed